Package Request: rbw

Description: rbw is a unofficial command line client for Bitwarden.
 
Official repo: https://git.tozt.net/rbw/about/
GitHub mirror: https://github.com/doy/rbw

If anyone wants to tackle this they can, I don't have the time or experience to do so currently.

```
error[E0658]: const generics are unstable
 --> /host/cargo/registry/src/github.com-1ecc6299db9ec823/arrayvec-0.7.0/src/utils.rs:6:15
  |
6 | impl<T, const N: usize> MakeMaybeUninit<T, N> {
  |               ^
  |
  = note: see issue #74878 <https://github.com/rust-lang/rust/issues/74878> for more information
```

--- a/srcpkgs/rbw/template
+++ b/srcpkgs/rbw/template
@@ -1,0 +1,23 @@
+# Template file for 'rbw'
+pkgname=rbw
+version=1.4.3
+revision=1
+build_style="cargo"
+build_helper="qemu"
+depends="pinentry"
+short_desc="Unofficial command line client for Bitwarden"
+maintainer="Omar Zeghouani <omarzeghouanii@gmail.com>"
+license="MIT"
+homepage="https://git.tozt.net/rbw"
+changelog="https://git.tozt.net/rbw/plain/CHANGELOG.md"
+distfiles="https://git.tozt.net/rbw/snapshot/rbw-${version}.tar.gz"
+checksum=2738aa6e868bf16292fcad9c9a45c60fe310d2303d06aea7875788bacda9b15b
+
+post_install() {
+	for shell in bash fish zsh; do
+		vtargetrun target/${RUST_TARGET}/release/rbw gen-completions $shell > rbw.$shell
+		vcompletion rbw.$shell $shell
+	done
+
+	vlicense LICENSE
+}

--- a/srcpkgs/rofi-rbw/template
+++ b/srcpkgs/rofi-rbw/template
@@ -1,0 +1,20 @@
+# Template file for 'rofi-rbw'
+pkgname=rofi-rbw
+version=1.0.0
+revision=1
+build_style=python3-pep517
+make_install_target="rofi_rbw-${version}-py3-none-any.whl"
+hostmakedepends="python3-setuptools python3-wheel"
+depends="python3 python3-ConfigArgParse rbw"
+short_desc="Rofi frontend for Bitwarden"
+maintainer="Omar Zeghouani <omarzeghouanii@gmail.com>"
+license="MIT"
+homepage="https://github.com/fdw/rofi-rbw"
+changelog="https://raw.githubusercontent.com/fdw/rofi-rbw/HEAD/CHANGELOG.md"
+distfiles="https://github.com/fdw/rofi-rbw/archive/${version}.tar.gz"
+checksum=25cebb7d0cbff19cd0dc54834455de0ae291c74c4d51d66a5c10a57093b49799
+
+post_install() {
+	vman src/rofi_rbw/docs/rofi-rbw.1
+	vlicense LICENSE
+}

